CodingChallenge/Views/SearchResultCell.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  SearchResultCell.swift
+//  CodingChallenge
+//
+//  Created by Edgar Barocio on 6/23/19.
+//  Copyright © 2019 Edgar Barocio. All rights reserved.
+//
+
+import UIKit
+
+class SearchResultCell: UICollectionViewCell {
+
+    @IBOutlet weak var albumArt: UIImageView!
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var artistLabel: UILabel!
```

Comment:
Centering these labels in the nib might look a little nicer.

Reply:
I actually decided agains centering this ones, on the writeup I am delivering I make the case than the result can be expanded, it currently displays two pieces of info and artwork, it has plenty more metadata included

Reply:
Also, the weak keyword isn't required here.